Полупанова Анна Андреевна, HW1

polupanovaanna opened the pull request.

polupanovaanna committed:
HW1 final

IlyaEp commented:
1. У Вас можно получить "05". Проверьте еще сток
2. Нет полноты
3. Ок
4. Ок
5. Ок

polupanovaanna committed:
Исправление домашней работы #1 по ФЯ.

Исправлены первые два задания про автоматы - исправление приложено отдельным файлом.

IlyaEp commented:
1,2. Ок
